mailing_christmas/controllers/generate_christmas_invoice.py

```diff
@@ -0,0 +1,134 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2016 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emanuel Cino <ecino@compassion.ch>
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+import logging
+
+from openerp import http, fields
+from openerp.tools import mod10r
+from openerp.http import request
+
+
+_logger = logging.getLogger(__name__)
+
+
+OK_RESULTS = {
+    'en_US': 'We successfully received your authorization. The amount will '
```

Comment:
@davidwul We should validate the result messages. Are you ok with English and French? Then we can ask for German and Italian translation. Is it ok to say the debit will be done in the next 3 days?